portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,28 +39,47 @@
 public class DataServlet extends HttpServlet {
 
   private static final String COMMENT_ENTITY_KEY = "Comment";
-  private static final String COMMENT_PROPERTY_NAME = "comment";
-  private static final String TIME_PROPERTY_NAME = "dateTime";
+  private static final String COMMENT_PROPERTY_NAME = "userComment";
+  private static final String TIME_PROPERTY_NAME = "dateTimeCreated";
```

Comment:
I agree that `"dateTimeCreated"` is a better name than `"dateTime"`--but why change `"comment"`?